onnx/backend/test/case/node/rmsnormalization.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import numpy as np
+
+import onnx
+from onnx.backend.test.case.base import Base
+from onnx.backend.test.case.node import expect
+
+
+# RMS normalization's reference implementation
+def _rms_normalization(X, W, axis=-1, epsilon=1e-5):  # type: ignore
+    shape = X.shape
+    rank = len(shape)
+    if axis < 0:
+        # If axis = -1 and rank of X is 4,
+        # the axis is changed to -1 + 4 = 3,
+        # which means the last axis.
+        axis = axis + rank
+
+    # Parameter used to convert N-D tensor RMS normalization to equivalent 2-D matirx operations.
+    row_number = np.prod(shape[:axis]).astype(np.int64)
+    col_number = np.prod(shape[axis:]).astype(np.int64)
+
+    # After reshaping input tensor X into a matrix,
+    # RMS normalization is equivalent to conducting
+    # standardization on each column vector (s.t. each
+    # column has zero mean and unit variance).
+    x_mat = np.reshape(X, (row_number, col_number))
+    # This computes RMS for every x_mat's column.
+    x_squared = np.power(x_mat, 2)
+    x_squared_mean = np.sum(x_squared, axis=1, keepdims=True) / col_number
```

Comment:
Can't we use `np.mean` here?

Reply:
Also: `np.mean` supports a tuple of axes, so: wonder if we can just specify `range(axis, len(shape))` as the axes, without having to reshape it twice. Not a big deal, but just to keep it simple.